internal/pkg/cli/env_init.go

```diff
@@ -329,12 +329,11 @@ func (o *initEnvOpts) validateCustomizedResources() error {
 		return fmt.Errorf("cannot import or configure vpc if --%s is set", defaultConfigFlag)
 	}
 	if o.importVPC.isSet() {
-		// We allow 0 or 2+ public subnets.
+		// Allow passing in VPC without subnets, but error out early for too few subnets-- we won't prompt the user to select more of one type if they pass in any.
 		if len(o.importVPC.PublicSubnetIDs) == 1 {
-			return fmt.Errorf("at least two public subnets must be imported to enable Load Balancing")
+			log.Warningf("At least two public subnets must be imported to enable Load Balancing.\n")
 		}
-		// We require 2+ private subnets.
-		if len(o.importVPC.PrivateSubnetIDs) < 2 {
+		if len(o.importVPC.PrivateSubnetIDs) == 1 {
```

Comment:
```suggestion
		if n := o.importVPC.PrivateSubnetIDs; n > 0 && n < 2 {
```

Reply:
Discussed offline; leaving as is bc 1 is the only int between 0 and 2.